pkg/transform/transform.go

```diff
@@ -323,6 +349,29 @@ func (f *frontMatterTransformer) TransformFrontMatter(ctx mdformatter.SourceCont
 
 func (f *frontMatterTransformer) Close(mdformatter.SourceContext) error { return nil }
 
+func (f *backMatterTransformer) TransformBackMatter(ctx mdformatter.SourceContext) ([]byte, error) {
+	b := bytes.Buffer{}
+	if err := f.b._template.Execute(&b, struct {
+		Origin MatterOrigin
+		Target MatterTarget
+	}{
+		Origin: f.origin,
+		Target: f.target,
+	}); err != nil {
+		return nil, err
+	}
+
+	// Add demarkation to back matter.
+	demark := "\n\n---\n"
+	m := b.Bytes()
+	m = append([]byte(demark), m...)
+
+	// Back matter is not parsed as YAML since it is generally assumed to be a block of text.
```

Comment:
This might be confusing that one is yaml, one not, maybe worth to document it better.